pkg/scaffold/controller/controller.go

```diff
@@ -158,19 +160,26 @@ func add(mrg manager.Manager, r reconcile.Reconcile) error {
 	return nil
 }
 
+var _ reconcile.Reconciler = &Reconcile{{ .Resource.Kind }}{}
+
 // Reconcile{{ .Resource.Kind }} reconciles a {{ .Resource.Kind }} object
 type Reconcile{{ .Resource.Kind }} struct {
-	client client.Client
+	client.Client
+	scheme *runtime.Scheme
 }
 
 // Reconcile reads that state of the cluster for a {{ .Resource.Kind }} object and makes changes based on the state read
 // and what is in the {{ .Resource.Kind }}.Spec
 // TODO(user): Modify this Reconcile function to implement your Controller logic.  The scaffolding writes
 // a Deployment as an example
+{{ if .Resource.CreateExampleReconcileBody -}}
```

Comment:
Is this a new field? What flag should we use in `kubebuilder` to match this field?

Reply:
This is new in controller-tools because controller-tools scaffolds out a whole example which kubebuilder does not.  Maybe keep the flags the same?